app/views/partners/PartnerTelephoneNumber.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+
+@this(
+        layout: layouts.layout,
+        errorSummary: components.errorSummary,
+        h1: components.h1,
+        form: FormWithCSRF,
+        inputText: components.inputText,
+        button: components.button
+)
+
+@(partnerTelephoneForm: Form[String], index: Int, name: Option[String])(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@layout(pageTitle = Some(title(partnerTelephoneForm, name.map(entityName => messages("partner.telephoneNumber.heading", entityName)).getOrElse(""))), backLink = true) {
```

Comment:
Entity won't be without name at this point I believe. Is default heading required in case?